### ISSUE TITLE
508 Issues from pa11y-rails

### ISSUE THREAD

**Issue by nickbristow** [18F/API-All-the-X#214]
Results can be recreated using [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/)

## Results for https://pages.18f.gov/API-All-the-X

* __Error__ This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.3:1. Recommendation: change text colour to #0476d2.
 * WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
 * `html > body > div > aside > a`
 * `<a class="skip-link visuallyhidden focusable" href="#main">Skip to Main Content</a>`

* __Error__ The html element should have a lang or xml:lang attribute which describes the language of the document.
 * WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2
 * `html`
 * `<html><head>
        <meta charset="u...</html>`

### Summary

__Errors:__ 2
